general/our-software.md

```diff
@@ -0,0 +1,98 @@
+requirement; it is also known as MoSCoW prioritization or MoSCoW analysis.
+
+This becomes vital when planning the development schedule of a project. Knowing what the customer needs over wants quickly
+becomes important when it comes to stretching their budget as far as it will go. They might *want* a rear view camera
+in their car, but they should probably have the engine first.
+
+#### Make it quickly
+There is a certain art to building software to a budget, and building solid software at that. Often developers can find
+that they want to move onto the less fundamental and less boring part of a project to focus on some interesting logic, 
+or that they want to see a view scaffolded completely with the must-have tasks **and** the nice-to-haves.
+
+No matter what you do - **PLEASE, PLEASE** refrain from doing this. There are times when you will be building some repetitive
+CRUD functionality or building some static views that don't have much going on, but most of the time these are the most
+important to the user journey/experience. This ensures that the user will have the most complete *flow* at all times
+throughout the project
+
+##### Minimum Viable Product (MVP)
+We see this work more commonly in an MVP methodology, where you write the least amount of code
+to satisfy a story point. This enables us to get the most functionality in the smallest amount of time, push that
+functionality out to the end users for review and move on, knowing whether it aligns with the users thoughts or not.
+
+This also means that we get any nasty surprises or hiccups as soon as possible throughout the timeline, which helps
+project managers understand where time might be saved, or where we may need some more resource to bring in the deadline.
+
+This is usually explained best in a graphic.
+
+![Planned Timeline](images/our-software/original-timeline.png)
+
+We can see in the above image that it seems all of the features that the client wants will fit into the timeline. The
+stars have aligned and this will be the perfect project and everyone will be happy. Great, right?
+
+![Actual Timeline](images/our-software/actual-timeline.png)
+
+Unfortunately we all know that's not how things plan out, and with just a few minor issues that extend a few of the tasks,
+the original deadline comes, and goes, and we don't even have all of the must have features in yet, meaning we cannot
+deploy this - we are forced to make up the different and the project is late. 
+
+![Rearranged Timeline](images/our-software/rearranged-timeline.png)
+
+With just a little bit of planning we can actually re-arrange the order in which we do things. Tasks might take slightly
+longer here or there but it's okay because the project is already at a state where it is **more** than deliverable at the
+deadline, meets the clients budgets and means they can get using the software - without compromising the company to work
```

Comment:
Should be `they can start using the software` instead of `they can get using the software`.